### PR TITLE
Moving list of approvers from .env to GitHub team

### PR DESCRIPTION
**Pull request by mutantcornholio**
Alongside the feature, following changes were made:
* Using `opstooling-integrations` for GitHub interations
* Upgraded `probot`, in order to align underlying `octokit` versions
between `opstooling-integrations` and `probot` itself
* Upgraded `opstooling-js` for the same reason

### CODE REVIEW

File: src/bot.ts [hunk 1]
```diff
@@ -51,35 +62,56 @@ const onIssueComment = async (state: State, context: IssueCommentCreatedContext,
...
+    void onIssueComment(state, context.payload, tipRequester, octokitInstance).then(respondOnResult, respondOnResult);
   });
 };
 
+process.env.PRIVATE_KEY = Buffer.from(envVar("PRIVATE_KEY_BASE64"), "base64").toString();
+
+// Aligning environment between Probot and opstooling-integrations
+process.env.GITHUB_APP_ID = process.env.APP_ID;
+process.env.GITHUB_AUTH_TYPE = "app";
+process.env.GITHUB_PRIVATE_KEY = process.env.PRIVATE_KEY;
```
Comment (mordamax) [inline src/bot.ts:115]: Im curious - could it be that we will need several types of auth? how would setting env look like?
i felt several times during review of opstooling integrations and here that it'd maybe be ideal if auth won't be dependent of env vars, but on some other configuration. Either passed by parameters or some config class wrapper. 
Just leaving thoughts, not a request to an action

**Review by mordamax — APPROVED** [review]

**Comment by mordamax** [comment]
probably missing the updates to helm?
like https://github.com/paritytech/substrate-tip-bot/blob/master/helm/values-parity-stg.yaml#L10

**Review by mordamax — CHANGES_REQUESTED** [review]
probably missing the updates to helm?
like https://github.com/paritytech/substrate-tip-bot/blob/master/helm/values-parity-stg.yaml#L10

**Comment by mutantcornholio** [comment]
> probably missing the updates to helm?
like https://github.com/paritytech/substrate-tip-bot/blob/master/helm/values-parity-stg.yaml#L10

Yeah

**Comment by mutantcornholio** [comment]
> probably missing the updates to helm? like https://github.com/paritytech/substrate-tip-bot/blob/master/helm/values-parity-stg.yaml#L10

I initially wanted to put it in secrets, but that info is by no means secret, so hardcoding it into the helm config seems viable

**Pull request merged**